app/controllers/api/v3/top_profiles_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V3
+    class TopProfilesController < ApiController
```

Comment:
we need a `skip_before_action :load_context` here, as this endpoint does not have a mandatory context paramete

Reply:
👍